site/docs/guide/games.md

```diff
@@ -61,11 +61,11 @@ const keyboard = new InlineKeyboard().game("Start my_game");
 // Notice that we have used game() unlike a normal inline keyboard
 // where we use url() or text()
 
-// Via replyWithGame method
+// Via the `replyWithGame` method
```

Comment:
```suggestion
// Via the `replyWithGame` method.
```

Reply:
This is not a sentence, no need for fullstop.

Reply:
This is not a sentence, no need for fullstop.